docs/advanced/classes.rst

```diff
@@ -490,6 +492,26 @@ you could equivalently write:
 
 which will invoke the constructor in-place at the pre-allocated memory.
 
+Brace initialization
+--------------------
+
+``pybind11::init<>`` internally uses C++11 brace initialization to call the
+constructor of the target class. This means that it can be used to bind
+*implicit* constructors as well:
+
+.. code-block:: cpp
+
+    struct POD {
```

Comment:
Not a POD (`std::string` is not trivially copyable). `Aggregate` should be the term for this kind of structure and brace initialization.